bolt-simple-taproot.md

```diff
@@ -717,11 +717,10 @@ The new output has the following form:
     * `to_delay_script_root = tapscript_root([to_delay_script])`
     * `to_delay_script` is the delay script:
         ```
-        to_self_delay
-        OP_CHECKSEQUENCEVERIFY
-        OP_DROP
         <local_delayedpubkey>
         OP_CHECKSIGVERIFY
+        to_self_delay
+        OP_CHECKSEQUENCEVERIFY
```

Comment:
The counter-party shouldn't allow low to_self_delay to be used, unless there's some trust perhaps. But I don't think allowing the potential of an unspendable script is worth the risk

Reply:
is it the case that `0 OP_CHECKSEQUENCEVERIFY` is unspendable? I don't see that from https://github.com/bitcoin/bips/blob/master/bip-0112.mediawiki#summary

Reply:
It's unspendable based on the context. With the new script change from @antonilol, the 0 doesn't get dropped as it previously would, so execution terminates with it and fails.

Reply:
oh right

Reply:
I think we should not allow a csv delay of 0 as it would break the cpfp anchor semantics, the local party will have 2 outputs spendable in the mempool and not only the anchor

Reply:
I'd prefer to not have to worry about this when implementing it. At the end of the day it is just 1 WU and I would personally rather leave that on the table than adding a zero check that:
- somebody could change in the future
- is incorrectly implemented

Reply:
yeah agree to that, will change it

Reply:
https://github.com/Roasbeef/lightning-rfc/compare/fe5a2898e66a1ff350e7c84bb0a8664dfd9c6a7c..c69660821e78d92b2778a74b3eb94e4dc086500d

Reply:
> I think we should not allow a csv delay of 0 

The delay of 1 is effectively a static parameter, if I understand the context.